modules/terraform/aws/main.tf

```
@@ -1,13 +1,14 @@
 locals {
-  region = lookup(var.json_input, "region", "us-east-1")
-  run_id = lookup(var.json_input, "run_id", "123456")
+  region        = lookup(var.json_input, "region", "us-east-1")
+  run_id        = lookup(var.json_input, "run_id", "123456")
+  creation_time = var.json_input["creation_time"]
```

```suggestion
  creation_time = lookup(var.json_input, "creation_time", null )
```
Can we use the lookup syntax here also. With default value as current time incase if the user doesn't pass the time